### ISSUE TITLE
Rename Codelyzer rule from `banana-in-a-box` to `banana-in-box`

### ISSUE THREAD

**Issue by kuzivany** [angular-eslint/angular-eslint#134]
The [Codelyzer rule, `template-banana-in-box`](http://codelyzer.com/rules/template-banana-in-box/) is incorrectly named `banana-in-a-box` in [`@angular-eslint/eslint-plugin-template`](https://github.com/angular-eslint/angular-eslint/blob/37e4cb20269fa184c379823780ed43c2dba17db6/packages/eslint-plugin-template/src/rules/banana-in-a-box.ts).

When using `template-banana-in-box` with TSLint, the `.eslintrc.js` file created by [converting to ESLint](https://github.com/typescript-eslint/tslint-to-eslint-config#usage) generates `@angular-eslint/template/banana-in-box` as a rule. This currently throws an error:

```log
1:1  error  Definition for rule '@angular-eslint/template/banana-in-box' was not found  @angular-eslint/template/banana-in-box
```

The [rules list](https://github.com/angular-eslint/angular-eslint#functionality) also refers to the rule as `template-banana-in-box`.

I propose renaming the file and rule in [`@angular-eslint/eslint-plugin-template`](https://github.com/angular-eslint/angular-eslint/tree/37e4cb20269fa184c379823780ed43c2dba17db6/packages/eslint-plugin-template/src/rules) to `banana-in-box` so it conforms to how it's referred to elsewhere

**Comment by kuzivany** [comment]
A quick fix in the meantime after using `tslint-to-eslint-config` is to rename the rule in your generated `.eslintrc.js`:

```diff
-        "@angular-eslint/template/banana-in-box": "error",
+        "@angular-eslint/template/banana-in-a-box": "error",
```

**Comment by JamesHenry** [comment]
Fix was merged and released as part of `v0.4.0-beta.1`
